src/ClientBuilder.php

```diff
@@ -252,9 +239,13 @@ public function setSdkVersion(string $sdkVersion): ClientBuilderInterface
      * @param string $packageName The package name that will be used to get the version from (i.e. "sentry/sentry")
      *
      * @return $this
+     *
+     * @deprecated since version 2.2, to be removed in 3.0
      */
     public function setSdkVersionByPackageName(string $packageName): ClientBuilderInterface
     {
+        @trigger_error(sprintf('Method %s() is deprecated since version 2.2 and will be removed in 3.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
You are still able to set both the SDK identifier and version using the appropriate methods `setSdkIdentifier` and `setSdkVersion`. The only thing this method did was to automate retrieving the version of a package using the `PrettyVersions` class, which can be done from the user so that we can simplify a bit our code